### DISCUSSION TITLE
frenchATC-for-MSFS2020 How to make it work ?

### DISCUSSION THREAD

**Discussion by MarcRouj** [Nash115/frenchATC-for-MSFS2020#30] [Help] [answered]
I would like to use frenchATC-for-MSFS2020. My problem is that I don't know how to use the "atc.py" file with python 3

I want a step-by-step explanation of the commands I need to write in PowerShell

Regards,
Marc

**Comment by Nash115** [comment, marked as answer]
- Vous pouvez lancer le programme en faisant un "double clic" dessus, ou alors en faisant un "Clic droit" > "Ouvrir avec" > "Python"

 - Sinon, dans le powershell, vous pouvez taper la commande `py atc.py` après vous être déplacé dans dans le dossier où se trouve le contenu de tout les fichiers du programme.
Par exemple, si le programme se trouve dans 'C:\Users\USER\Desktop\atc', vous pourrez exécuter les commandes suivantes :
```
cd C:\Users\USER\Desktop\atc
py atc.py
```

**Comment by MarcRouj** [comment]
Hello ! Thanks for answering me ! I actually posted this question, and a few hours later I remembered how to use PowerShell.
The manipulation was completely successful, and I was able to test your ATC
Regards,
Marc
________________________________
De : nash115 ***@***.***>
Envoyé : mercredi 6 décembre 2023 22:22
À : Nash115/frenchATC-for-MSFS2020 ***@***.***>
Cc : MarcRouj ***@***.***>; Author ***@***.***>
Objet : Re: [Nash115/frenchATC-for-MSFS2020] frenchATC-for-MSFS2020 How to make it work ? (Discussion #30)


  *   Vous pouvez lancer le programme en faisant un "double clic" dessus, ou alors en faisant un "Clic droit" > "Ouvrir avec" > "Python"

  *   Sinon, dans le powershell, vous pouvez taper la commande py atc.py après vous être déplacé dans dans le dossier où se trouve le contenu de tout les fichiers du programme.
Par exemple, si le programme se trouve dans 'C:\Users\USER\Desktop\atc', vous pourrez exécuter les commandes suivantes :

cd C:\Users\USER\Desktop\atc
py atc.py
